snapshotter/service_integ_test.go

```diff
@@ -104,15 +100,14 @@ func launchContainerWithRemoteSnapshotterInVM(ctx context.Context, vmID string)
 				AllowMMDS: true,
 				CNIConfig: &proto.CNIConfiguration{
 					NetworkName:   "fcnet",
-					InterfaceName: "veth0",
+					InterfaceName: fmt.Sprintf("veth%s", vmID),
```

Comment:
While I don't believe having each microVM on the same interface causes any issue, there also isn't a reason to have them on the same interface so just place each VM on it's own interface.